lib/jellyfish/room.ex

```diff
@@ -58,13 +58,16 @@ defmodule Jellyfish.Room do
 
   def registry_id(room_id), do: {:via, Registry, {Jellyfish.RoomRegistry, room_id}}
 
-  @spec start(Config.t()) :: {pid(), id()}
+  @spec start(Config.t()) :: {:ok, pid(), id()}
   def start(config) do
-    id = UUID.uuid4()
+    id = config.room_id || UUID.uuid4()
```

Comment:
I think this should be done in Config.from_params so that here, we always have room id already assigned